orchestra/tests/test_dashboard.py

```diff
@@ -483,13 +486,15 @@ def test_assignment_snapshot_upgrades(self):
                     'type': TaskAssignment.SnapshotType.SUBMIT,
                 }, {
                     'data': {},
-                    'datetime': datetime.utcnow().isoformat(),
+                    'datetime': (
+                        datetime.utcnow().replace(tzinfo=tzutc()).isoformat()),
                     'type': TaskAssignment.SnapshotType.SUBMIT,
                     'work_time_seconds': 1
                 }]}
         upgraded_snapshots = deepcopy(old_snapshots)
-        upgraded_snapshots['__version'] = 1
+        upgraded_snapshots['__version'] = 2
         upgraded_snapshots['snapshots'][0]['work_time_seconds'] = 0
+        upgraded_snapshots['snapshots'][0]['datetime'] += '+00:00'
```

Comment:
I think that a timezone-aware datetime will output the '+offset' with `isoformat()`, but that's worth checking. If not, I guess this is fine.